### ISSUE TITLE
Main documentation link is wrong.

### ISSUE THREAD

**Issue by jthomas** [IBM-Bluemix-Docs/openwhisk#140]
From this page: https://cloud.ibm.com/openwhisk/ 

In the left-hand menu, under the "Getting Started" drop-down, the link for documentation is 
https://cloud.ibm.com/docs/openwhisk?topic=cloud-functions-getting_started#getting-started-with-openwhisk and it should be https://cloud.ibm.com/docs/openwhisk?topic=cloud-functions-getting-started#getting-started-with-openwhisk

**Comment by derekpoindexter** [comment]
Thanks for the feedback @jthomas.

When we restructured the TOC, we changed several file names (which breaks the links). For now, we've built temporary redirects which is why you see the "this page has moved" banner.

I've got an issue open with the core doc framework team. 
https://github.ibm.com/Bluemix/Bluemix-doc-framework/issues/2573#issuecomment-11479255

And one specifically for the getting started page.
https://github.ibm.com/Bluemix/Bluemix-doc-framework/issues/2593

Once the redirects are in place you will no longer see the "this page have moved" banner.
